watertap/unit_models/zero_order/electrocoagulation_zo.py

```diff
@@ -0,0 +1,611 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021-2023, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+###############################################################################
+
+"""
+This module contains a zero-order representation of an electrocoagulation unit.
+"""
+
+from pyomo.environ import (
+    Var,
+    Param,
+    Constraint,
+    Expression,
+    units as pyunits,
+)
+from pyomo.common.config import ConfigValue, In
+
+from idaes.core import declare_process_block_class
+from idaes.core.util.constants import Constants
+from idaes.core.util.misc import StrEnum
+from watertap.core import build_sido, ZeroOrderBaseData
+
+
+class ElectrodeMaterial(StrEnum):
+    aluminum = "aluminum"
+    iron = "iron"
+
+
+class ReactorMaterial(StrEnum):
+    pvc = "pvc"
+    stainless_steel = "stainless_steel"
```

Comment:
Is ReactorMaterial being used by this model? Maybe I'm missing it, but I don't see where this is actually utilized elsewhere in the code.